### PR TITLE
ROR-16 Connect Scheduling API to Hackney DRS

### PR DESCRIPTION
**Pull request by Bhavesh-Patel**
This also applies changes for:

- [ROR-51](https://hackney.atlassian.net/browse/ROR-51)
- [ROR-52](https://hackney.atlassian.net/browse/ROR-52)
- [ROR-53](https://hackney.atlassian.net/browse/ROR-53)

### CODE REVIEW

File: HousingRepairsSchedulingApi/Services/Drs/DrsService.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ namespace HousingRepairsSchedulingApi.Services.Drs
 
     public class DrsService : IDrsService
     {
-        private const string DrsContract = "0";
+        private const string DrsContract = "H01";
```
Comment (maysakanoni) [inline HousingRepairsSchedulingApi/Services/Drs/DrsService.cs:13]: what does H01 represent?

File: HousingRepairsSchedulingApi/serverless.yml [hunk 2]
```diff
@@ -26,6 +26,7 @@ functions:
     name: ${self:service}-${self:provider.stage}
     handler: HousingRepairsSchedulingApi::HousingRepairsSchedulingApi.LambdaEntryPoint::FunctionHandlerAsync
     role: lambdaExecutionRole
+    timeout: 30
```
Comment (maysakanoni) [inline HousingRepairsSchedulingApi/serverless.yml:29]: why does this need to be set to 30 seconds?

File: HousingRepairsSchedulingApi/Services/Drs/DrsService.cs [hunk 3]
```diff
@@ -13,7 +13,7 @@ public class DrsService : IDrsService
         private const string DrsContract = "H01";
         private const string DummyPrimaryOrderNumber = "HousingRepairsOnlineDummyPrimaryOrderNumber";
         private const string DummyUserId = "HousingRepairsOnlineUserId";
-        private const string Priority = "Priority 20 Days";
+        private const string Priority = "I";
```
Comment (maysakanoni) [inline HousingRepairsSchedulingApi/Services/Drs/DrsService.cs]: I think your changing this to an N but i will approve

**Review by maysakanoni — DISMISSED** [review]

File: HousingRepairsSchedulingApi/Services/Drs/DrsService.cs (hunk 3 above)
Comment (Bhavesh-Patel) [reply HousingRepairsSchedulingApi/Services/Drs/DrsService.cs]: Yes, just updated priority to 'N'

File: HousingRepairsSchedulingApi/serverless.yml (hunk 2 above)
Comment (Bhavesh-Patel) [reply HousingRepairsSchedulingApi/serverless.yml:29]: This is to allow multiple requests to be sent to DRS when searching for available appointments ([Timing reference from Lincoln](https://lbhackney-it.github.io/housing-repairs-online/spikes/scheduled-repairs#appendix-a)).

File: HousingRepairsSchedulingApi/Services/Drs/DrsService.cs (hunk 1 above)
Comment (Bhavesh-Patel) [reply HousingRepairsSchedulingApi/Services/Drs/DrsService.cs:13]: "H01" represents a general building repair and is the contract value the PO (Lindsay Matthews) said should be used.

**Review by jamiejordanmt — DISMISSED** [review]

**Review by maysakanoni — DISMISSED** [review]

**Review by maysakanoni — APPROVED** [review]

**Pull request merged**